tests/authorization.spec.ts

```diff
@@ -0,0 +1,45 @@
+import {describe} from "mocha";
+import { Browser, Builder, By, Key, until, WebDriver } from "selenium-webdriver"
+import {expect} from "chai";
+
+describe('[SHARELANE-2] Authorization', async () => {
+    it('should authorize user', async () => {
+        let driver: WebDriver = await new Builder().forBrowser(Browser.CHROME).build();
+        try {
+            await driver.get('https://www.sharelane.com/cgi-bin/register.py')
+            const continueButton = await driver.findElement(By.css('input[value="Continue"]'))
+            const zipCode = await driver.findElement(By.name('zip_code'))
+            await zipCode.sendKeys('11111')
+            await continueButton.click()
+
+            const registerButton = await driver.findElement(By.css('input[value="Register"]'))
+            await driver.findElement(By.name('first_name')).sendKeys('qqqqq')
+            await driver.findElement(By.name('last_name')).sendKeys('qqqqq')
+            await driver.findElement(By.name('email')).sendKeys('qqq@aaa.com')
+            await driver.findElement(By.name('password1')).sendKeys('11111')
+            await driver.findElement(By.name('password2')).sendKeys('11111')
+            await registerButton.click()
+
+            let successRegistrationMessage = await driver.findElement(By.className('confirmation_message'))
+            expect(await successRegistrationMessage.getText()).to.equal('Account is created!')
+
+            const emailElement = await driver.findElement(By.css('html > body > center > table > tbody > tr:nth-of-type(6) > td > table > tbody > tr:nth-of-type(4) > td > table > tbody > tr:nth-of-type(1) > td:nth-of-type(2) > b'))
```

Comment:
I guess this locator could be simplified to: By.name('email')

Reply:
<img width="1350" alt="email css" src="https://github.com/user-attachments/assets/4fbbdbd9-8eed-44ae-b87c-3a3a54f9c90b">


It doesn't work

Reply:
Please build an xpath locator with following-sibling logic for that element as I showed today

Reply:
Done